src/HomeWork5/Tictactoe.java

```diff
@@ -0,0 +1,151 @@
+            for (int j = 0; j < b; j++) {
+                System.out.print(tTc[i][j] + " ");
+            }
+            System.out.println();
+        }
+        char choice;
+        char[][] game = {{'1', '2', '3'},
+                {'4', '5', '6'},
+                {'7', '8', '9'}};
+
+
+        System.out.println("Поиграем в игру :");
+        System.out.println("Сверху поле дла игры");
+        System.out.println("Что бы поставить X или 0\nнужно выбрать нужную цифру");
+        System.out.println("Да победит сильнейший!!!!");
+        System.out.println("Чтобы начать нажмите 1 ");
+        Scanner scanner = new Scanner(System.in);
+        int fScanner = scanner.nextInt();
+        if (fScanner == 1) {
+            int random = (int) (Math.random() * 2);
+            if (random % 2 == 0) {
+                System.out.println("Игрок №2 ходит первым\n");
+            } else {
+                System.out.println("Игрок №1 ходит первым\n");
+            }
+        } else {
+            System.out.println("Ввидите 1!");
+        }
+        Scanner scanner1 = new Scanner(System.in);
+        int gameX = 0;
+        int choiceGame;
+
+
+        while (true) {
+            choiceGame = scanner1.nextInt();
+            if (gameX % 2 == 0){
+                choice = 'x';
+                System.out.println("Очередь x");
+            }
+            else{
+                choice = '0';
+                System.out.println("Очередь 0");
+            }
+
+            printField(game);
+            if ( choiceGame <= 9 && choiceGame > 0) {
+                switch (choiceGame) {
+                    case 1:
+                        game[0][0] = choice;
+                        break;
+                    case 2:
+                        game[0][1] = choice;
+                        break;
+                    case 3:
+                        game[0][2] = choice;
+                        break;
+                    case 4:
+                        game[1][0] = choice;
+                        break;
+                    case 5:
+                        game[1][1] = choice;
+                        break;
+                    case 6:
+                        game[1][2] = choice;
+                        break;
+                    case 7:
+                        game[2][0] = choice;
+                        break;
+                    case 8:
+                        game[2][1] = choice;
+                        break;
+                    case 9:
+                        game[2][2] = choice;
+                        break;
+                }
+            } else {
+                System.out.println("Введите число из оставшихся на поле!");
+                continue;
+
+            }
+
+            checkingForMatch(game);
+
+            if (boolGame) {
+                System.out.println("YOU WIN!!!");
+                printField(game);
+                break;
+            }
+            gameX ++;
+
+        }
+    }
+
+
+
+
```

Comment:
delete unnecessary spaces

Reply:
I fixed tictactoe in this branch. Can you check

Reply:
commit bugFix